utils/doclint/cli.js

```diff
@@ -84,6 +85,15 @@ async function run() {
       });
       writeAssumeNoop(filePath, content, dirtyFiles);
     }
+
+    // Patch pom.xml
+    {
+      const introPath = path.join(PROJECT_DIR, 'docs', 'src', 'intro-java.md');
+      const pomVersionRe = new RegExp('^(\\s*<artifactId>playwright<\\/artifactId>\\n\\s*<version>)(.*)(<\\/version>)$', 'gm');
+      let content = fs.readFileSync(introPath).toString();
+      content = content.replace(pomVersionRe, '$1' + playwrightVersion + '$3');
```

Comment:
Patch version in java may be behind (we don't always roll patch releases into ports) so I'd rather update it to `major.minor.0`

Reply:
Updated thanks!